translations/Base_Dutch.lang

```diff
@@ -1,133 +1,133 @@
+mysqlPane.test.hostname: "Hostnaam: **%**"
+mysqlPane.test.maximumPoolSize: "Maximale poolgrootte: **%**"
+mysqlPane.test.minimumIdle: "Minimaal inactief: **%**"
+mysqlPane.test.password: "Wachtwoord: **%**"
+mysqlPane.test.port: "Port: **%**"
+mysqlPane.test.ssl: "SSL: **%**"
+mysqlPane.test.title: "Test wijzigingen"
+mysqlPane.test.username: "Gebruikersnaam: **%**"
+mysqlPane.username: "gebruikersnaam"
+mysqlSetup.actionBar: "Standaard: **%**"
+mysqlSetup.subtitle: "Typ **%** in de Chat"
+mysqlSetup.title: "MySQL-installatie"
+none: "geen"
+pageableView.basicSearch.action: "Klik om te zoeken"
+pageableView.basicSearch.title: "Zoeken"
+pageableView.next.action: "Klik om naar de volgende pagina te gaan"
+pageableView.next.title: "Volgende"
+pageableView.previous.action: "Klik om naar de vorige pagina te gaan"
+pageableView.previous.title: "Vorige"
+pageableView.search.closeAction: "Klik om de zoekopdracht te **sluiten**"
+pageableView.search.subtitle: "Typ een zoekterm"
+pageableView.search.title: "Zoek"
+restartView.button.action: "Klik om de server opnieuw te starten"
+restartView.button.title: "Herstarten"
+restartView.restarting: "Opnieuw opstarten..."
+restartView.title: "Start de server opnieuw op"
+second: "seconde"
+seconds: "seconden"
+settings: "instellingen"
+settingsView.select.action: "Klik om Instellingen weer te geven"
+settingsView.select.indicator: "Deze pagina is momenteel **geselecteerd**"
+settingsView.title: "Instellingen > %"
+timePickerView.addDay: "Voeg 1 Dag toe"
+timePickerView.addHour: "Voeg 1 Uur toe"
+timePickerView.addMinute: "Voeg 1 Minuut toe"
+timePickerView.addSecond: "Voeg 1 Seconde toe"
+timePickerView.confirm.action: "Klik om te bevestigen"
+timePickerView.confirm.title: "Bevestigen"
+timePickerView.decrementAction: "Klik om te verwijderen"
+timePickerView.incrementAction: "Klik om toe te voegen"
+timePickerView.removeDay: "Verwijder 1 Dag"
+timePickerView.removeHour: "Verwijder 1 Uur"
+timePickerView.removeMinute: "Verwijder 1 Minuut"
+timePickerView.removeSecond: "Verwijder 1 Seconde"
+update.clickToAuthenticate: "Klik op de onderstaande link om te verifiëren met **Discord**:"
+update.remote.notify: "Upload van **%from%** naar **%to%**"
+update.serverOffline: "Er is een fout opgetreden bij het communiceren met onze updateserverr"
+update.updatedPlugin: "Plugin bijgewerkt naar **%**, herladen.."
+update.youAreNotVerified: "U bent niet geverifieerd op onze **Discord**"
+update.youNotOwnPlugin: "U bent niet de eigenaar van deze plug-in. **Koop het** om deze functie te ontgrendelen"
+updateView.no.action: "Klik om op deze versie te blijven"
+updateView.no.desc.newVersion: "Nieuwe versie: **%**"
+updateView.no.desc.yourVersion: "Jou versie: **%**"
+updateView.no.title: "Blijf op deze versie"
+updateView.title: "Bijwerken van %from% naar %to%"
+updateView.yes.action: "Klik om **Bijwerking** te laden"
+updateView.yes.desc: "Deze functie vereist ** Discord ** om uw aankoop te verifiëren"
```

Comment:
Is there a specific reason why you added spacing before and after the `Discord`?

Reply:
Oh no that's ecedental